packages/test-helpers/src/bin/github-comment-manager.test.js

```diff
@@ -0,0 +1,181 @@
+/* global describe it expect */
```

Comment:
what does this comment mean?

Reply:
Its used by lint to set global variables